bot/code_review_bot/tasks/base.py

```diff
@@ -31,12 +31,28 @@ def run_id(self):
 
     @property
     def name(self):
+        """Short name used to identify the task"""
         return self.task["metadata"].get("name", "unknown")
 
+    @property
+    def display_name(self):
+        """
+        Longer name used to describe the task to humans
+        By default fallback to short name
+        """
+        return self.name
+
     @property
     def state(self):
         return self.status["state"]
 
+    def build_help_message(self, files):
+        """
+        An optional help message aimed at developers to reproduce the issues detection
+        A list of relative paths with issues is specified to build a precise message
+        By default it's empty (None)
+        """
```

Comment:
We could make display_name and help_message optional values of the standard JSON format that checkers have to provide.
Otherwise, we'll never be able to be fully generic and get rid of the custom AnalysisTask classes.

Reply:
:arrow_right: #339 

I'll do that as a followup, with documentation.